engine/apps/user_management/models/organization.py

```diff
@@ -75,6 +73,9 @@ def get_queryset(self):
         return OrganizationQuerySet(self.model, using=self._db).filter(deleted_at__isnull=True)
 
 
+# TODO: in a subsequent PR, remove the inheritance from MaintainableObject (plus generate the database migration file)
+# this will remove the maintenance related columns that're no longer used on the organization object
+# class Organization(models.Model):
```

Comment:
these columns will be removed in a subsequent PR, for now I just removed any reference to them. I believe if we remove them in this PR, it'd cause a brief period of HTTP 500s where Organization serializers in pods running the old version fail to find these columns in the database (because they were just dropped).